CHANGELOG.md

```diff
@@ -1,5 +1,24 @@
 # Change Log
 
+## [0.2.0]()
```

Comment:
What the deal with this syntax?  This is essentially an empty (invalid) URL.

Reply:
yeah that is weird. I have no idea why I did that. maybe it should link to the GitHub release page

Reply:
fixed